modules/plugin/coverage-multidim/netcdf/pom.xml

```diff
@@ -108,15 +108,8 @@
       <artifactId>commons-cli</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-      <version>1.6.4</version>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>log4j-over-slf4j</artifactId>
-      <version>1.6.4</version>
+      <groupId>org.apache.logging.log4j</groupId>
```

Comment:
I was just talking with @dromagnoli the other day about this dependency, in the context of the GeoServer NetCDF module... there is seems not to be required after all. Wondering if here it's the same... you could try kicking it away and see if the module builds anyways.

Reply:
That would be *so* much help.

I think it is just saying it needs sl4j api to be available (and hosting application should be able to choose what sl4j uses).

Reply:
Reading https://docs.unidata.ucar.edu/netcdf-java/current/userguide/using_netcdf_java_artifacts.html indicates they need at least one jar providing slf4j api. 

I think the least damaging approach would be to use the slf4j-jdk14 dependency for testing, and indicate in our docs that netCDF requires SL4J matching your environment.